cdap-app-templates/cdap-etl/cdap-data-streams-base/src/main/java/io/cdap/cdap/datastreams/DataStreamsPipelineSpecGenerator.java

```diff
@@ -28,17 +29,18 @@
 import org.apache.hadoop.fs.Path;
 
 import java.util.Set;
+import javax.annotation.Nullable;
 
 /**
  * Generates specs for data stream pipelines.
  */
 public class DataStreamsPipelineSpecGenerator
   extends PipelineSpecGenerator<DataStreamsConfig, DataStreamsPipelineSpec> {
 
-  <T extends PluginConfigurer & DatasetConfigurer> DataStreamsPipelineSpecGenerator(T configurer,
-                                                                                    Set<String> sourcePluginTypes,
-                                                                                    Set<String> sinkPluginTypes) {
-    super(configurer, sourcePluginTypes, sinkPluginTypes, Engine.SPARK);
+  <T extends PluginConfigurer & DatasetConfigurer> DataStreamsPipelineSpecGenerator(
```

Comment:
Please keep the same spacing we had before, it made this constructor more readable.

Reply:
Can't do that, `@Nullable RuntimeConfigurer runtimeConfigurer` will make the line exceed 120 characters